package.json

```
@@ -86,8 +86,8 @@
     "@carbon/layout": "10.7.1",
     "@carbon/motion": "10.6.0",
     "@carbon/themes": "10.10.3",
-    "carbon-components": "10.10.3",
-    "carbon-components-react": "7.10.3",
+    "carbon-components": "^10.12.0",
```

These should be pinned to specific version.
```suggestion
    "carbon-components": "10.12.0",
```